------------------------------------------------------------------------
r1247 | hodgestar | 2014-02-16 19:32:21 +0100 (So, 16. Feb 2014) | 1 Zeile

Disable the speedups C extension on CPython >= 3.3 since Genshi doesn't support the new Unicode C API yet.
------------------------------------------------------------------------
Index: setup.py
===================================================================
--- a/setup.py	(Revision 1246)
+++ b/setup.py	(Revision 1247)
@@ -65,9 +65,13 @@
 
 
 if Feature:
+    # Optional C extension module for speeding up Genshi:
+    # Not activated by default on:
+    # - PyPy (where it harms performance)
+    # - CPython >= 3.3 (the new Unicode C API is not supported yet)
     speedups = Feature(
         "optional C speed-enhancements",
-        standard = not is_pypy,
+        standard = not is_pypy and sys.version_info < (3, 3),
         ext_modules = [
             Extension('genshi._speedups', ['genshi/_speedups.c']),
         ],
